src/KafkaSchemaRegistryApiClient.php

```diff
@@ -229,6 +230,21 @@ public function getLatestSubjectVersion(string $subjectName): ?string
         return $schemaVersions[$lastKey];
     }
 
+    /**
+     * @param string $mode
+     * @return bool
+     */
+    public function setImportMode(string $mode): bool
+    {
+        try {
+            $this->httpClient->call('PUT', 'mode/', ['mode' => $mode]);
```

Comment:
```suggestion
            $result = $this->httpClient->call('PUT', 'mode/', ['mode' => $mode]);
            return $result === json_encode(['mode' => $mode]);
```
also don't catch the exception